signer/src/transaction_coordinator.rs

```diff
@@ -678,12 +673,20 @@ where
             });
 
         tracing::info!("broadcasing bitcoin transaction");
-
+        // Broadcast the transaction to the Bitcoin network.
         self.context
             .get_bitcoin_client()
             .broadcast_transaction(&transaction.tx)
             .await?;
 
+        // Publish the transaction to the P2P network so that peers get advance
+        // knowledge of the sweep.
+        self.send_message(
```

Comment:
Tbh I'm not sure which is better, publishing this before or after the bitcoin broadcast.. :thinking: